deployment/docker-compose.yml

```diff
@@ -38,6 +38,8 @@ services:
   influxdb:
     container_name: influxdb
     image: influxdb:latest
+    volumes:
+      - "./influxdb:/var/lib/influxdb2"
```

Comment:
Add to the readme, that the directory needs to be created for the DB data.

Reply:
Sorry I didn't get this one. Can you please clarify

Reply:
Put in the README.md that for the deployment you need to create that influxdb folder.